### DISCUSSION TITLE
Automatic light and dark mode selectin according to system settigs

### DISCUSSION THREAD

**Discussion by peterschmidler** [arthelokyo/astrowind#465] [Ideas]
It would be nice if light and dark mode would change automatically according to the system settings just as with AstroPaper.

**Comment by Harold-Anderson** [comment]
@peterschmidler In src/config.yaml, if you scroll to the bottom and set ui: theme: to 'system', it will do that.

![image](https://github.com/onwidget/astrowind/assets/5002519/360ef9be-943e-40a1-980e-76b80459d476)
